packages/coil-extension/src/background/services/ActiveTabLogger.ts

```diff
@@ -28,4 +31,18 @@ export class ActiveTabLogger {
       }
     })
   }
+
+  sendLogEvent(message: string) {
```

Comment:
What is the point of all of this? The background page has its own log.

Reply:
will remove it, just an easy way of viewing everything in one tab